go.mod

```diff
@@ -419,6 +420,7 @@ replace (
 	// TODO: Remove me when Podman uses a pinned runc version:
 	// https://github.com/containers/podman/blob/74aa681e59352257eb5f25f089dffa66d6345a3a/go.mod#L78
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.1-0.20220617142545-8b9452f75cbc
+	github.com/opencontainers/runtime-tools => github.com/opencontainers/runtime-tools v0.0.0-20221026201742-946c877fa809
```

Comment:
No, it's not you missing something. It's me missing that since we don't need to divert any more the NRI repo to a forked one, there is also no point in using a related replace for runtime-tools, as instead of making things clearer it just makes them more confusing. Removed replace and updated runtime-tools version in require.